build.sbt

```diff
@@ -1,10 +1,12 @@
 import scoverage.ScoverageKeys
 import uk.gov.hmrc.DefaultBuildSettings.integrationTestSettings
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+import uk.gov.hmrc.DefaultBuildSettings.targetJvm
 
 val appName = "import-control-wsdls"
 
 val silencerVersion = "1.7.11"
+targetJvm := "jvm-11"
```

Comment:
I don't think we need to mention this targetJvm post 2nd of May and we will loose backward compatibility.

Reply:
Ok, thanks. I'll close this PR then.